content/en/cloudcraft/getting-started/crafting-better-diagrams.md

```diff
@@ -0,0 +1,78 @@
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-account.png" alt="Screenshot highlighting the dropdown to select an account to want to scan in the Cloudcraft interface with a red arrow pointing to the dropdown." responsive="true" style="width:100%;">}}
+
+Under **Region**, select the regions you want to scan. By default, `Global` and your default region are selected, but you can click the **More** button to select or search for additional regions.
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-region.png" alt="Screenshot highlighting the regions to scan in the Cloudcraft interface with red arrows pointing to the regions and the More button." responsive="true" style="width:100%;">}}
+
+After selecting, regions are scanned automatically and the number of resources found is displayed right next to the region name. You can click the **Sync** button above the **Region** section to trigger a manual scan of all selected regions.
+
+<div class="alert alert-info">Note that selecting many regions may impact performance of the live scanning process.</div>
+
+## Filter resources
+
+You can filter resources by type and tags.
+
+Tags are automatically detected from your AWS account and displayed in the **Custom tags**, **AWS tags**, **Terraform tags**, and **Kubernetes tags** sections.
+
+- **Custom tags** are tags that you added to resources in AWS or Azure.
+- **AWS tags** are tags that are automatically added to resources by AWS.
+- **Terraform tags** are tags that are automatically added to resources by Terraform.
+- **Kubernetes tags** are tags that are automatically added to resources by Kubernetes.
+
+To filter resources by type, click the **Resource** section and select the resource type you want to filter by. By default, all resource types are selected and displayed in order of the number of resources found.
+
+The same goes for tags. Click the **Custom tags**, **AWS tags**, **Terraform tags**, or **Kubernetes tags** section and select the tags you want to filter by. By default, all tags are selected and displayed in order of the number of resources found, with `Untagged` always at the bottom.
+
+<div class="alert alert-info">Consider focusing on the most relevant resoure types and tags for your specific use case to ensure optimal performance and readability of your diagram.</div>
+
+Say you want to create a diagram showing only EC2 instances and RDS databases. Click the **Resource** section, deselect all resource types, and select only `EC2` and `RDS`.
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-specific-resources.mp4" alt="A 9 seconds video showing a Cloudcraft user selecting EC2 and RDS instances from the Resource section." video="true">}}
+
+Click **Apply layout** to create a diagram showing only the selected resources.
```

Comment:
```suggestion
```suggestion

## Use cases

### Create a diagram showing only EC2 instances and RDS databases

1. Click the **Resource** section.
1. Deselect all resource types, and select **EC2** and **RDS**.
1. Click **Apply layout** to create a diagram showing only the selected resources.

{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-specific-resources.mp4" alt="A 9 seconds video showing a Cloudcraft user selecting EC2 and RDS instances from the Resource section." video="true">}}
```